new-site/src/docs/elements/design-tokens/colour/components/brand-colour-combinations-table.tsx

```diff
@@ -0,0 +1,32 @@
+  { colourNameEn: <b>Bus</b>, textColourToken: <code>--color-white</code>, textContrast: <span><b>AAA</b> <code>12 : 1</code></span>, example: <ContrastExample color="var(--color-white)" background="var(--color-bus)">AAA</ContrastExample> },
+  { colourNameEn: <b>Summer</b>, textColourToken: <code>--color-black-90</code>, textContrast: <span><b>AAA</b> <code>11,1 : 1</code></span>, example: <ContrastExample color="var(--color-black-90)" background="var(--color-summer)">AAA</ContrastExample> },
+  { colourNameEn: <b>Fog</b>, textColourToken: <code>--color-black-90</code>, textContrast: <span><b>AAA</b> <code>10 : 1</code></span>, example: <ContrastExample color="var(--color-black-90)" background="var(--color-fog)">AAA</ContrastExample> },
+  { colourNameEn: <b>Copper</b>, textColourToken: <code>--color-black-90</code>, textContrast: <span><b>AAA</b> <code>9,4 : 1</code></span>, example: <ContrastExample color="var(--color-black-90)" background="var(--color-copper)">AAA</ContrastExample> },
+  { colourNameEn: <b>Suomenlinna</b>, textColourToken: <code>--color-black-90</code>, textContrast: <span><b>AAA</b> <code>9,1 : 1</code></span>, example: <ContrastExample color="var(--color-black-90)" background="var(--color-suomenlinna)">AAA</ContrastExample> },
+  { colourNameEn: <b>Gold</b>, textColourToken: <code>--color-black-90</code>, textContrast: <span><b>AAA</b> (AAA for large text) <code>7,1 : 1</code></span>, example: <ContrastExample color="var(--color-black-90)" background="var(--color-gold)">AAA</ContrastExample> },
+  { colourNameEn: <b>Brick</b>, textColourToken: <code>--color-white</code>, textContrast: <span><b>AA</b> (AAA for large text) <code>6,1 : 1</code></span>, example: <ContrastExample color="var(--color-white)" background="var(--color-brick)">AA</ContrastExample> },
+  { colourNameEn: <b>Metro</b>, textColourToken: <code>--color-black-90</code>, textContrast: <span><b>AA</b> (AAA for large text) <code>5,2 : 1</code></span>, example: <ContrastExample color="var(--color-black-90)" background="var(--color-metro)">AA</ContrastExample> },
+  { colourNameEn: <b>Coat of Arms</b>, textColourToken: <code>--color-white</code>, textContrast: <span><b>AA</b> (AAA for large text) <code>5 : 1</code></span>, example: <ContrastExample color="var(--color-white)" background="var(--color-coat-of-arms)">AA</ContrastExample> },
+  { colourNameEn: <b>Tram</b>, textColourToken: <code>--color-white</code>, textContrast: <span><b>AA</b> (AAA for large text) <code>4,6 : 1</code></span>, example: <ContrastExample color="var(--color-white)" background="var(--color-tram)">AA</ContrastExample> },
+];
+
+export default function BrandColourCombinationsTable() {
+  return (
+    <Table dense cols={cols} rows={rows} indexKey="id" renderIndexCol={false} />
```

Comment:
This can be simplified to:

`<Table dense cols={cols} rows={rows} indexKey="colourNameEn" />`